### PR TITLE
Small fixes to image URLs

### PR DESCRIPTION
**Pull request by tagawa**
Also fixed hyphen/commas.

### CODE REVIEW

**Comment by tagawa** [comment]
Content only so self-merging.
/cc @jbarrett

File: views/email/a/1_common.tx [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
+    That's why it's important to use privacy alternatives that don't share your personal information, such as <a href="https://www.duckduckgo.com/">DuckDuckGo</a> for search.
 </p>
 
 <p style="margin-bottom: 1em">
     Using Incognito mode to keep you private online is kind of like using a bucket to put out a raging fire:<br /><br />
     <center><img src="https://duckduckgo.com/assets/email/fire.jpg" alt="Picture of man putting out fire with small bucket of water" width="200" height="200" style="padding-bottom:15px;" /></center>
 </p>
 <p style="margin-bottom: 1em">
-   If you didn’t know that private browsing isn’t private — you’re not alone! <a href="https://spreadprivacy.com/is-private-browsing-really-private/">67% of people</a> who use private browsing <i>over-estimate</i> the protection that private browsing modes offer.
+   If you didn’t know that private browsing isn’t private, you’re not alone! <a href="https://spreadprivacy.com/is-private-browsing-really-private/">67% of people</a> who use private browsing <i>over-estimate</i> the protection that private browsing modes offer.
```
Comment (jbarrett) [inline views/email/a/1_common.tx]: ',' needs a space I think.

Comment (tagawa) [reply views/email/a/1_common.tx]: There is a space, so it's `private, you're`

**Comment by tagawa** [comment]
Not self-merging after all, so will leave this open for merging later or I can make a new PR in a few days if necessary.

File: views/email/a/1_common.tx (hunk 1 above)
Comment (jbarrett) [reply views/email/a/1_common.tx]: So there is - looks like diff presentation is messed up.

:+1:

Comment (jbarrett) [reply views/email/a/1_common.tx]: We'll need to look at this again - whatever sort of space this is, it's not recognised by Gecko :)

**Pull request merged**